### PR TITLE
[flutter_tools] remove fuchsia tooling

### PR DESCRIPTION
**Pull request by jonahwilliams**
Fixes https://github.com/flutter/flutter/issues/123040

### CODE REVIEW

**Review by zanderso — APPROVED** [review]

File: .ci.yaml [hunk 1]
```diff
@@ -523,15 +523,6 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Linux fuchsia_precache
```
Comment (dnfield) [inline .ci.yaml:526]: So we can delete the code in the engine builders that uploads this stuff?

What about the code that builds it?

**Review by dnfield — APPROVED** [review]
LGTM. I thought CI would start failing on this but neat.

File: .ci.yaml (hunk 1 above)
Comment (jonahwilliams) [reply .ci.yaml:526]: Not sure about engine builders, probably no since google3 artifacts still roll through flutter.

Comment (jonahwilliams) [reply .ci.yaml:526]: the tooling itself isn't used though

**Comment by dnfield** [comment]
I guess the internal failures can just get patched to not depend on this code?

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by chinmaygarde** [comment]
Seems good to go but not sure about the Google testing failure. Are we blocked on an investigation?

**Comment by jonahwilliams** [comment]
Blocked on investigating g3 linkage

**Comment by dnfield** [comment]
Should we re-open https://github.com/flutter/flutter/pull/123084 and land it in the mean time to fix FFX problems for people?

**Comment by jonahwilliams** [comment]
Yes please!

**Comment by christopherfujino** [comment]
@jonahwilliams I'm guessing we're still blocked?

**Comment by jonahwilliams** [comment]
Sorry, I need to follow up on how this is being used in g3. In the meantime, Dan landed a fix that makes sure this is disabled if the user has not opted in. I'm going to close this for now

**Pull request closed without merging**